src/gradualizer_db.erl

```diff
@@ -26,6 +26,11 @@
 
 %% Types for the Erlang Abstract Format
 -type type() :: gradualizer_type:abstract_type().
+-type typed_record_field() :: {typed_record_field,
```

Comment:
Does it matter that the type uses a tuple instead of the equivalent record?

Reply:
Which record? This is an abstract form. They are defined as tuples, not records.

This change fixed the dialyzer error btw.